documentation/library_guide/pstl/macros.rst

```diff
@@ -32,61 +32,68 @@ Additional Macros
-                                  When the macro is set to 0 there is no dependency on
-                                  the oneAPI DPC++/C++ Compiler and runtime libraries.
-                                  Trying to use DPC++ policies will lead to compilation errors.
---------------------------------- ------------------------------
-``ONEDPL_ALLOW_DEFERRED_WAITING`` This macro allows waiting for completion of certain algorithms executed with 
-                                  DPC++ policies to be deferred. (Disabled by default.)
---------------------------------- ------------------------------
-``ONEDPL_FPGA_DEVICE``            Use this macro to build your code containing oneDPL parallel
-                                  algorithms for FPGA devices. (Disabled by default.)
---------------------------------- ------------------------------
-``ONEDPL_FPGA_EMULATOR``          Use this macro to build your code containing Parallel STL
-                                  algorithms for FPGA emulation device. (Disabled by default.)
+                                   Using this macro may have the same effect on the implementation of parallel
+                                   algorithms in the C++ standard libraries of GCC and LLVM.
+---------------------------------- ------------------------------
+``ONEDPL_USE_TBB_BACKEND``         This macro controls the use of oneAPI Threading Building Blocks (oneTBB) or
+                                   Threading Building Blocks (TBB) for parallel policies.
+                                   When the macro is set to 0, algorithms with the ``par`` and ``par_unseq`` policies are only
+                                   executed by the calling thread. This is recommended for code that should not depend on the
+                                   presence of the oneTBB or TBB library. When the macro is not defined (by default)
+                                   or evaluates to a non-zero value,
+                                   parallel policies are executed using the oneTBB or TBB library.
+---------------------------------- ------------------------------
+``ONEDPL_USE_DPCPP_BACKEND``       This macro enables the use of the DPC++ policies.
+                                   When the macro is not defined (by default)
+                                   or evaluates to non-zero, DPC++ policies are enabled.
+                                   When the macro is set to 0 there is no dependency on
+                                   the oneAPI DPC++/C++ Compiler and runtime libraries.
+                                   Trying to use DPC++ policies will lead to compilation errors.
+---------------------------------- ------------------------------
+``ONEDPL_USE_PREDEFINED_POLICIES`` This macro enables the use of predefined policies objects,
+                                   e.g. ``dpcpp_default``, ``dpcpp_fpga``. When the macro is not defined (by default)
```

Comment:
Let's fuse the sentences starting with "when".
```rst
When the macro is not defined (by default) or evaluates to non-zero, predefined policies objects and make functions without arguments, e.g. ``make_device_policy()``, ``make_fpga_policy()``, can be used.
```

Reply:
I found that the initial wording is simpler, because it implies the way how to turn the corresponding objects off in the second "when" sentence.